pixel-saver@deadalnix.me/buttons.js

```diff
@@ -48,7 +48,11 @@ function createButtons() {
 	let orders = order.replace(/ /g, '').split(':');
 	
 	orders[0] = orders[0].split(',');
-	orders[1] = orders[1].split(',');
+	
+	// Checking the secondary button orders
+	if(typeof orders[1] != 'undefined') orders[1] = orders[1].split(',');
+	// if doesn't exist then use the default orders
+	else orders[1] = orders[0];
```

Comment:
oh, okay :D so that's why I have a close button on the right as well.
the `!(1 in orders)` will throw error if the orders is undefined, I mean if some sort of miracle delete the orders variable then it will break also the `(1 in orders)` will do a for loop to check all of the keys which way pointing the exact key is faster.